common/src/main/java/com/wynntils/utils/render/RenderUtils.java

```diff
@@ -833,7 +833,6 @@ public static void renderVignetteOverlay(PoseStack poseStack, CustomColor color,
                 Texture.VIGNETTE.height());
 
         RenderSystem.setShaderColor(1, 1, 1, 1);
-        RenderSystem.enableBlend();
```

Comment:
O_o

Why did we ever do something like this? (And why did it not work?)

Reply:
I am not sure.. most likely a typo. This is why render types are better code wise, porting to them is longer though..